### PR TITLE
Strict settings on the NathanGain fine control modifier.

### PR DESCRIPTION
**Pull request by oldgalileo**

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 1]
```diff
@@ -5,48 +5,94 @@
 import org.usfirst.frc4904.standard.commands.chassis.ChassisShift;
 import org.usfirst.frc4904.standard.humaninput.Driver;
 import org.usfirst.frc4904.standard.subsystems.chassis.SolenoidShifters;
+import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.SpeedModifier;
+import edu.wpi.first.wpilibj.command.Command;
 
-public class NathanGain extends Driver {
+strictfp public class NathanGain extends Driver {
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: We only have one platform, so I am not sure strictfp is needed.

Comment (oldgalileo) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: Darn. I've been foiled.

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 2]
```diff
@@ -13,40 +15,84 @@
...
+	
 	@Override
 	public void bindCommands() {
-		RobotMap.Component.xbox.a
-			.whenPressed(new ChassisShift(RobotMap.Component.chassis.getShifter(), SolenoidShifters.ShiftState.DOWN));
-		RobotMap.Component.xbox.b
-			.whenPressed(new ChassisShift(RobotMap.Component.chassis.getShifter(), SolenoidShifters.ShiftState.UP));
+		RobotMap.Component.xbox.y.toggleWhenPressed(new EnableFineModifier(modifier));
+		RobotMap.Component.xbox.a.whenPressed(new ChassisShift(RobotMap.Component.chassis.getShifter(), SolenoidShifters.ShiftState.DOWN));
+		RobotMap.Component.xbox.b.whenPressed(new ChassisShift(RobotMap.Component.chassis.getShifter(), SolenoidShifters.ShiftState.UP));
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: Which of the formats for the shift lines is correct under the new formatter?

**Comment by oldgalileo** [comment]
Tested. Works beautifully as is. Issue is that it too rapidly changes the speed (instantaneously) and could damage the bot.

**Review by c0d3rman — APPROVED** [review]

**Pull request merged**